pkg/apis/internal.acorn.io/v1/appinstance.go

```diff
@@ -185,6 +190,7 @@ type AppInstanceStatus struct {
 type Defaults struct {
 	Volumes map[string]VolumeDefault `json:"volumes,omitempty"`
 	Memory  map[string]*int64        `json:"memory,omitempty"`
+	Region  string                   `json:"region,omitempty"`
```

Comment:
Do we need to default Region concept in app too? Or it can just be `status.Region` and it will be calculated through spec.region, and fallback to project.default.region when spec.region is empty?

Reply:
This is the calculation of the default region based on the project default. The idea here is that by storing the default region on the app status, the region won't change for the app if an admin changes the default region for the project.

We don't want the app to be changed unless something in the app changes, not because something external (like the project default region) changed.